src/index.ts

```diff
@@ -450,6 +450,34 @@ export class Graph<N extends InputNode, L extends InputLink> {
     return node && this.points.getNodeRadius(node)
   }
 
+  /**
+   * Track multiple node positions by their ids.
+   * @param ids Array of nodes ids.
+   */
+  public trackNodesByIds (ids: string[]): void {
+    this.points.trackNodesByIds(ids)
+  }
+
+  /**
+   * Track multiple node positions by their indices.
+   * @param ids Array of nodes indices.
+   */
+  public trackNodesByIndices (indices: number[]): void {
+    this.points.trackNodesByIds(
+      indices.map(index => this.graph.getNodeByIndex(index))
+        .filter((d): d is N => d !== undefined)
+        .map(d => d.id)
+    )
+  }
+
+  /**
+   * Get current X and Y coordinates of the tracked nodes.
+   * @returns Map where keys are the ids of the nodes and values are corresponding `[number, number]` with X and Y coordinates of the node.
```

Comment:
@Stukova  

A `Map` object where keys are the ids of the nodes and values are their corresponding X and Y coordinates in the `[number, number]` format.